### ISSUE TITLE
How do I fix confidence 0%? 

### ISSUE THREAD

**Issue by KarinaNi** [andresavic/AmongKey#36]
<img width="361" alt="image" src="https://github.com/andresavic/AmongKey/assets/32227493/de2724a7-188b-4b3b-9586-43ab8b27f0ae">

How do i fix this error?

**Comment by Kskz** [comment]
Same problem for me

**Comment by MunjalFaneesh** [comment]
did you find a fix?

**Comment by GWBrett28** [comment]
Hi, I know this is a bit off topic but I am new to this. How do I even add mods like this onto the Among-Us application on Mac? It's all wrapped up for me when I click "show package contents".

**Comment by ninjvz** [comment]
i need support with this too please

**Comment by tx620** [comment]
same here

**Comment by tx620** [comment]
just use bluestacks got it up and running in 10 minutes flat, this is outdated

**Comment by ninjvz** [comment]
which version and if you can send the link pls

**Comment by tx620** [comment]
Of course, [https://www.bluestacks.com/download.html](url), works very well, if you have trouble do reply. Works with other mobile games too, like brawl stars.

**Comment by tx620** [comment]
If needed I can send screenshots

**Comment by ninjvz** [comment]
Oh i havent seen this version yet, ill try and get back to you thank you so much.

**Comment by tx620** [comment]
Of course man

**Comment by ninjvz** [comment]
works like a breeze
